src/_test_/App.test.js

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import App from '../App';
+import store from '../Redux/configureStore';
+import '@testing-library/jest-dom';
+
+it('Check if Item component has changed', () => {
+  const tree = renderer
+    .create(
+      <Provider store={store}>
+        <BrowserRouter>
+          <App />
+        </BrowserRouter>
+      </Provider>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+it('Check if the App container is there', async () => {
+  render(
+    <Provider store={store}>
+      <BrowserRouter>
+        <App />
+      </BrowserRouter>
+    </Provider>,
+  );
+  const container = await screen.findByTestId('app-container');
+  expect(container).toBeInTheDocument();
+});
```

Comment:
- Your test related to the app is falling. please, kindly fix this. see the image below:


![Screenshot from 2023-02-17 23-43-31](https://user-images.githubusercontent.com/59397159/219811820-b5dd9c30-c3de-4671-bf7f-ebf5d14fccba.png)

Reply:
Hi @zieeco Thank you for your comment. I have resolved this error.